code/test-utils/build.gradle

```diff
@@ -41,6 +41,7 @@ dependencies {
     implementation project(':edge')
     implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
     implementation "androidx.test.ext:junit:${rootProject.ext.junitVersion}"
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.7'
```

Comment:
Just to confirm, as you are moving the files from module `:edge` to `:test-utils`, does the `:edge` project still require the `androidTestImplementation 'com.fasterxml.jackson.core:jackson-databind:2.12.7'` dependency or can it be removed?

Reply:
Cool! Can you make that change in `code/edge/build.gradle`?